proot-distro.sh

```diff
@@ -847,9 +847,10 @@ run_proot_cmd() {
 		case "$DISTRO_ARCH" in
 			aarch64) cpu_emulator_path="@TERMUX_PREFIX@/bin/qemu-aarch64";;
 			arm)
-				if [ "$DEVICE_CPU_ARCH" != "aarch64" ]; then
+				#TODO: improve detection, some aarch64 CPU don't support 32bit arm instuctions
```

Comment:
Please remove your todo comments and commented out code.
You can of course keep them locally, just make sure not to stage those hunks for final inclusion in the PR.

Reply:
Hey, I've adressed this issue in new commit. Brought back if's and added a condition for 32 bit support.